browser/ui/views/speedreader/speedreader_dancing_books.cc

```diff
@@ -0,0 +1,84 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/views/speedreader/speedreader_dancing_books.h"
+
+#include "brave/app/vector_icons/vector_icons.h"
+#include "ui/gfx/canvas.h"
+#include "ui/gfx/geometry/vector2d.h"
+#include "ui/gfx/paint_vector_icon.h"
+#include "ui/native_theme/native_theme.h"
+
+namespace {
+
+constexpr SkColor kBraveSpeedreaderGraphicLight =
+    SkColorSetRGB(0xE9, 0xEC, 0xEF);
+constexpr SkColor kBraveSpeedreaderGraphicDark =
+    SkColorSetRGB(0x49, 0x50, 0x57);
+
+constexpr int kMinimumWidth = 287;
+constexpr int kMinimumHeight = 61;
+
+}  // anonymous namespace
+
+namespace speedreader {
+
+SpeedreaderDancingBooks::SpeedreaderDancingBooks() : views::View() {
```

Comment:
don't need to call `View()` explicitely.

Reply:
removed. just did `= default` now